website/content/en/docs/best-practices/multi-tenancy.md

```diff
@@ -0,0 +1,162 @@
+      cpu: 10m
+      memory: 64Mi
+    ... 
+```
+
+In order to prevent tenants from consuming all the resources on a cluster or blocking other workloads 
+from being scheduled, many production clusters will define [ResourceQuota][k8s-resources-quotas] configuration.
+[ResourceQuota][k8s-resources-quotas] configuration in a namespace will cause all pods without 
+[Requests and limits][k8s-requests-limits] for `CPU` and `Memory` to be rejected.
+
+Because OLM-managed Operators do not run on Cluster infra nodes, they will consume paid CPU cores. 
+Users will naturally want to limit the maximum amount of resources these non-workload components can take up. 
+For this reason, if you are integrating with [OLM][olm], you should, **always ship your operator `Deployment` with resource requests amounts for CPU and memory**.
+
+The [ResourceQuota][k8s-resources-quotas] configuration also is applied to tenant workloads that are managed by your operator. Cluster administrators will typically set
+a [ResourceQuota][k8s-resources-quotas] for each tenant's namespace/project as part of the onBoarding. If your operator creates `Pods/Deployments` inside
+the tenant namespace which do not specify at least resource requests for `CPU` and `Memory`, those `Pods/Deployments` will fail to schedule and tenants
+will not be able to use your operator. Therefore, your operator should **always apply at least resource requests for CPU and
+memory** for `Pods/Deployments` that it creates as part of reconciliation. Ideally your operator also applies resource 
+limits to those `Pods/Deployments`. 
+
+#### General guidelines
+
+**Note that pretty much all controllers consume:**
+- `CPU`: largely based on the number of reconciliations they perform, which are generally related to event activity for resources they're watching.
+- `Memory`: largely based on the number of primary resources that exist (multiplied by some factor based on the number of operand resources they need to watch as a result) via informer caches.
+
+Operator authors cannot anticipate how they scale in usage in all customer environments, so setting one-size-fits-all 
+that fits well for all scenarios is not possible then, clusters administrators must be allowed to change the values 
+in order to respond to changes in their clusters’ needs, such as by resizing control plane nodes to provide more resources.
+
+**Following are some general recommendations to manage the resources:**
+
+- **MUST** declare resource requests for both CPU and memory.
+- **OPTIONALLY** setting the resources limit for Memory and CPU (Note that if your project is integrated with OLM then, admins would be able to customize it via the [subscription][olm-subscriptions]). Also, be aware that our project was built with Kubebuilder or SDK scaffold then for the `Operator/manager`(See `config/manager/manager.yaml`), then some values which allow you to get started are scaffolded by default, however, you ought to optimize based on the needs.
```

Comment:
Subscription is for operator configuration. It would be worth mentioning it and how it is done for operand even if it is trivial (Ok seeing that it is mentioned later in the doc: see my comment above regarding clearer separation of what is for operators and what is for operands)
Nitpicking: "be aware that our project was built with Kubebuilder or SDK scaffold...", proposing "be aware that if the project was generated by Kubebuilder or SDK scaffold then some values for the  `Operator/manager`(See `config/manager/manager.yaml`) are populated by default to get you started. However, you need to amend them based on your own tests and the specific needs of your operator"

Reply:
Tried to persuade this one.